src/main/java/org/spongepowered/api/event/message/MessageSinkEvent.java

```diff
@@ -47,14 +49,14 @@
      *
      * @return The message sink the message in this event will be sent to
      */
-    MessageSink getSink();
+    Optional<MessageSink> getSink();
```

Comment:
Wasn't there a discussion to keep this non-`Optional`?

https://github.com/SpongePowered/SpongeAPI/pull/980#discussion_r48228484

Or was there a side discussion I don't not know about?